docs/content/docs/application_requirements.md

```diff
@@ -7,4 +7,13 @@ type: docs
 # Application Requirements
 
 ## Application UIDs
-Do not run applications with the user ID (UID) value of **1500**. This is **reserved** for the Envoy proxy sidecar container injected into pods by OSM's sidecar injector.
\ No newline at end of file
+Do not run applications with the user ID (UID) value of **1500**. This is **reserved** for the Envoy proxy sidecar container injected into pods by OSM's sidecar injector.
+
+## Ports
+Do not use the following ports as they are used by the Envoy sidecar.
```

Comment:
```suggestion
The following ports are used by the Envoy sidecar.
```

Reply:
@draychev, I don't agree with the suggested wording, it is rather ambiguous when this section of the doc needs to be explicit.